<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrightNContrast</class>
 <widget class="QMainWindow" name="BrightNContrast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrightNContrast</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="OriginalView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="GenerateView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadOriginal">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Generate">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QSlider" name="BrightSlider">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ContrastSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>230</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>270</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="brightCheck">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="contrCheck">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
